examples/resources/incident_status/import.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Did we already support importing for all these other resources?

Reply:
Yes! Just hadn't put it in the docs 😭